pkg/manifest/operations.go

```diff
@@ -297,7 +297,14 @@ func (o *Operations) uninstall(deployInfo types.InstallInfo) (bool, error) {
 	return true, err
 }
 
-func (o *Operations) getManifestForChartPath(installInfo types.InstallInfo) *types.ParsedFile {
+func (o *Operations) GetManifestForChartPath(installInfo types.InstallInfo) *types.ParsedFile {
```

Comment:
why did you choose to create a new method instead of just adjusting the existing logic?

Reply:
The purpose here is just don't want to use `installInfo.DisableCache` flag to influence existing logic too much, because the `.renderSrc.GetRawManifest(installInfo)` is in the middle of the process, then it needs to use this flag to control the pre and post process, and it's actually not a new method, it use same signature of old method but use  `DisableCache` flag to control the further flow.